hack/verify-structured-logging.sh

```diff
@@ -46,9 +50,35 @@ done < <(echo "${all_packages[@]}" | tr " " "\n")
 
 ret=0
 echo -e "\nRunning structured logging static check on all packages"
-GOOS=linux    logcheck "${packages[@]}" || ret=$?
-GOOS=windows  logcheck "${packages[@]}" || ret=$?
+
+# Function to run logcheck and capture output
+run_logcheck() {
+  local os="$1"
+  shift
+  local output
+  output=$(GOOS="$os" logcheck "$@" 2>&1)
+  echo "$output"
```

Comment:
My 2 cents:

- remove this line here as for current code base, it reports a lot of violations. I may choose to resolve them in phases
- in Phase 1: treat violations with string "unstructured logging function" as fatal error, dump the problematic entries under L62 
- in Phase 2: fix all violations, and hence don't need to filter out "unstructured logging function"

Reply:
removed.

In addition, I added todo items to record these things 🤔